app/cdap/components/Administration/TetheringTabContent/Connections/ConnectionsTable.tsx

```diff
@@ -45,22 +46,15 @@ interface IConnectionsTableProps {
   renderLastColumn: (instanceName: string) => JSX.Element;
 }
 
-const ConnectionsTable = ({
-  tableData,
-  columnTemplate,
-  renderLastColumn,
-}: IConnectionsTableProps) => {
-  const transformedTableData = tableData.map((conn) => ({
-    gcloudProject: conn.metadata.metadata.project,
-    instanceName: conn.name,
-    region: conn.metadata.metadata.location,
-    allocationData: conn.metadata.namespaceAllocations,
-    status: conn.connectionStatus,
-  }));
-
-  const renderTableHeader = () => (
+const renderTableHeader = (tableData: IConnection[], columnTemplate: string) => {
```

Comment:
This and the function below have been moved up here to avoid re-declaration on every render.